dists/androidsdl/build.sh

```diff
@@ -9,7 +9,9 @@ if [ \! -d ../../../androidsdl ] ; then
    cd androidsdl
    git submodule update --init project/jni/iconv/src
    git checkout d378ee692f2e380a0ab0635c1df2eb6941b5bf58
-   cd ../scummvm/dists/androidsdl
+   cd project/jni/iconv/src
+   git checkout 07bead221ed4fa61cc8d880db3d9a5e704866097
```

Comment:
Why do you need this specific version of iconv?

Reply:
Because the build fails with the latest ones, and our build.sh script was already doing a checkout of a specific version of androidsdl (d378ee692f2e380a0ab0635c1df2eb6941b5bf58) which compiles ok with the specific version of iconv (07bead221ed4fa61cc8d880db3d9a5e704866097) and NDK r15c.
This is from thread:
https://forums.scummvm.org/viewtopic.php?f=17&t=14811

Reply:
Understood. Can you please add a comment here, for future reference?